### PR TITLE
Tweak the blog teaser image

### PR DESCRIPTION
**Pull request by stefansundin**
Hi!

I read the 2.6.2 release announcement in my RSS feed reader (I use inoreader.com). It looked a bit interesting to see that Chrome was warning about a Flash Player being included.

Screen Shot 2020-10-21 at 19 35 48

I took a look and saw that the KeePassXC logo svg file was being included with the `<object>` tag, and my feed reader is not including all of the attributes for whatever reason. But apparently the object tag is not really necessary for svg images these days. https://stackoverflow.com/questions/4476526/do-i-use-img-object-or-embed-for-svg-files

So I wanted to see if I could improve it. Unfortunately, I do not think just changing it to `` will be very good for most feed readers since the CSS stylesheet from the website isn't included (I don't think there's a good way to include it either). It would look like this:

Screen Shot 2020-10-21 at 19 56 10

Basically a big fat icon before any text. Not really an improvement.

So then I thought that the teaser image only makes a lot of sense on the list of blog posts on the website. It is not that necessary inside of the blog post itself. So I made the decision to just move it to front matter and only include it in `/blog/index.html`.

Please let me know what you think.

### CODE REVIEW

**Review by phoerious — APPROVED** [review]
I am fine with this change. Reduces the amount of copy-paste code for blog posts.
